.github/workflows/operator-ci.yml

```diff
@@ -26,10 +26,24 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+
+  conditional:
+    name: Check conditional workflows and jobs
+    if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak'
```

Comment:
Is there a specific reason we do not want to run this on forks? Or was this copied over from the codeql workflow by accident?

Reply:
We have a daily scheduled run, and don't want to run this on forks. This part isn't being changed in this PR, it's just moving where this check is, see the changes to the build job